engine_redis.go

```diff
@@ -1814,7 +1860,8 @@ func (s *shard) Channels() ([]string, error) {
 	if s.useCluster() {
 		return nil, errors.New("channels command not supported when Redis Cluster is used")
 	}
-	dr := newDataRequest(dataOpChannels, []interface{}{"CHANNELS", s.messagePrefix + "*"})
+
+	dr := newDataRequest(dataOpChannels, []interface{}{"CHANNELS", strings.Join([]string{s.messagePrefix, "*"}, "")})
```

Comment:
I think this change does not not make a lot of sense - it just makes things  more verbose